nit picks changes to ts files (#577)
huggingface/huggingface_hub: tasks/src/audio-classification/about.md

### Command Recognition

> 

Reviewer: ```suggestion
```

### Emotion recognition

> 

Reviewer: ```suggestion
```

### Speaker Identification

> 

Reviewer: ```suggestion
```

> Speaker Identification consists of classifying the audio of the person speaking in it. Speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.

Reviewer: I'd say:
Speaker Identification is of classifying the audio of the person speaking it. Speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.

Reviewer 2: Imo the proposed change from Omar looks good

> Speaker Identification consists of classifying the audio of the person speaking in it. Speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.

Reviewer: ```suggestion
Speaker Identification is classifying the audio of the person speaking. Speakers are usually predefined. You can try out this task with [this model](https://huggingface.co/superb/wav2vec2-base-superb-sid). A useful dataset for this task is VoxCeleb1.
```

## Solving audio classification for your own data

We have some great news! You can do fine-tuning (transfer learning) to train a well-performing model without requiring as much data. Pretrained models such as Wav2Vec2 and HuBERT exist. [Facebook's Wav2Vec2 XLS-R model](https://ai.facebook.com/blog/wav2vec-20-learning-the-structure-of-speech-from-raw-audio/) is a large multilingual model trained on 128 languages and with 436K hours of speech.

> We suggest reviewing the following [example](https://github.com/huggingface/transformers/tree/master/examples/pytorch/audio-classification) ([Colab Notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/master/examples/audio_classification.ipynb)) to learn how to fine-tune a model for audio classification with a single or multiple GPUs and share it on the Hub.

Reviewer: Review is I think a bit different, it's like auditing something.

> We suggest reviewing the following [example](https://github.com/huggingface/transformers/tree/master/examples/pytorch/audio-classification) ([Colab Notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/master/examples/audio_classification.ipynb)) to learn how to fine-tune a model for audio classification with a single or multiple GPUs and share it on the Hub.

Reviewer: ```suggestion
We suggest checking out the following [example](https://github.com/huggingface/transformers/tree/master/examples/pytorch/audio-classification) ([Colab Notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/master/examples/audio_classification.ipynb)) to learn how to fine-tune a model for audio classification with a single or multiple GPUs and share it on the Hub.
```
